bidiServer/server.js

```
@@ -113,12 +113,21 @@ function getErrorResponse(plainCommandData, errorCode, errorMessage) {
   };
 }
 
+async function sendClientMessage(message, connection) {
+  try {
+    const messageStr = JSON.stringify(message);
+    connection.sendUTF(messageStr);
+    console.log('Sent message: ' + messageStr);
+  } catch (e) {
+    console.log("sendClientMessage failed", e);
```

Would it be better to just let the exception be thrown and maybe catch it at the highest possible level? This should be unreachable code, right?

---

This code was added for internal debugging. Removed.